### PR TITLE
LRQA-39045 | Master

### PR DESCRIPTION
**Pull request by acbbennett**
https://issues.liferay.com/browse/LRQA-39045

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/macros/MessageboardsPortlet.macro [hunk 1]
```diff
@@ -308,7 +308,7 @@
 
 		<var name="key_siteName" value="${userScreenName}" />
 
-		<execute function="AssertTextEquals" locator1="SitePages#SITE_NAME" value1="${userScreenName}" />
+		<execute function="AssertTextEquals#assertPartialText" locator1="SitePages#SITE_NAME" value1="${userScreenName}" />
```
Comment (austinchiang) [inline portal-web/test/functional/com/liferay/portalweb/macros/MessageboardsPortlet.macro]: @acbbennett Can you change the title of this commit? You're not changing a locator here, but a function.

**Comment by acbbennett** [comment]
@austinchiang  Commit message changed per comment above, and rebased branch to head

**Comment by austinchiang** [comment]
No test changes needed.

**Comment by austinchiang** [comment]
Pushed to BChan.
https://github.com/brianchandotcom/liferay-portal/pull/56643

**Pull request closed without merging**